lib/matrix/projection.nit

```diff
@@ -150,4 +150,27 @@ redef class Matrix
 		var rotated = self * rotation
 		self.items = rotated.items
 	end
+
+	# Rotation matrix from Euler angles `pitch`, `yaw` and `roll` in radians
+	#
+	# Apply a composition of intrinsic rotations around the axes x-y'-z''.
+	# Or `pitch` around the X axis, `yaw` around Y and `roll` around Z,
+	# applied successively. All rotations follow the right hand rule.
+	#
+	# This service aims to respect the world axes and logic of `gamnit`,
+	# it may not correspond to all needs.
+	new gamnit_euler_rotation(pitch, yaw, roll: Float)
```

Comment:
It would be too general. There are many different Euler rotations, this is only x-y'-z'' as used by gamnit. For custom Euler rotations, clients should combine 3 calls to `rotate` by choosing the order and axes.